black-mulberry-data-transfer/src/main/java/com/black/mulberry/data/transfer/request/CategoryProductRequest.java

```diff
@@ -0,0 +1,15 @@
+package com.black.mulberry.data.transfer.request;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class CategoryProductRequest {
+    private String name;
+    private int categoryParentId;
```

Comment:
in database it is fixed as bigint in java it is long accordingly please change it to long